contracts/governance/GovernorParameters.sol

```diff
@@ -1,4 +1,17 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Flagging this change @cygnusv - did we have MIT there on purpose? I was sure we are going with GPL everywhere?

Reply:
That's a mistake 🙈

Reply:
So it's fixed now :)